### ISSUE TITLE
Chat received and read receipts

### ISSUE THREAD

**Issue by jlwatts** [keybase/keybase-issues#3101]
Definite feature request, but it would be great if chat could have received and read receipts like whatsapp.

**Comment by livejamie** [comment]
Would like this as well

**Comment by mikegchambers** [comment]
Is there any architectural reason this can't be done?  

Signal does this reasonably well and - especially with 1-on-1 chats - it's super helpful.   Slack (the closest user-functional equivalent to Keybase chat) does not.   

I realize that an exploding message can be 'exploded' before it's been read, and I'm ok with that, but is there a way that the platform can say if a person has 'seen' the message (exploding or otherwise).   Or is the intention for it to remain 'user-functional' equivalent to Slack (which is fine, if that's the aim).

Thanks

**Comment by programmergrammar** [comment]
I would love to see this in Keybase. Telegram does a great job of putting a little check mark by messages that the recipient has seen.

**Comment by antoszka** [comment]
This feature would bring Keybase closer as my go-to chat app as well. Are there major architectural blockers?

**Comment by pciavald** [comment]
to me keybase is the killer app of at least 5 apps I use except this feature

**Comment by sysfu** [comment]
Any action on this over the past two years? This feature would be helpful so you don't have to manually inquire if the other party has read your previous message(s) yet.

**Comment by sysfu** [comment]
You're absolutely right about that, a friend of mine showed me a commit graph post Zoom takeover and it was dismal. I hate buyouts and acquisitions, the founders never seem to care about what happens to the userbase that is PART OF THE VALUEATION of the companies they sell. Keybase founders sold us down the river and they suck.

**Comment by livejamie** [comment]
We uninstalled and moved on years ago, recommend you do the same
